### ISSUE TITLE
First load on a product page is broken. 

### ISSUE THREAD

**Issue by jorkvist** [DivanteLtd/vue-storefront#2279]
Browsing to a product page in incognito on an Android with latest chrome browser and the page is broken (missing product pictures and no links are working) . Suggest it's the same if not in incognito ie first load.

Tested with this product:
https://demo.vuestorefront.io/p/WS08/minerva-lumatech-trade-v-tee/WS08

**Comment by jorkvist** [comment]
Remote debug: 
![image](https://user-images.githubusercontent.com/5425824/51712098-07071580-202e-11e9-81af-b0b5b053c637.png)

**Comment by jorkvist** [comment]
Remote debug
![image](https://user-images.githubusercontent.com/5425824/51712234-6a914300-202e-11e9-8cb5-ece5f97dd004.png)

**Comment by filrak** [comment]
@pkarw is it solved?

**Comment by jorkvist** [comment]
@pkarw Are those fixes live on demo.vuestorefront.io? I still have the issue on that instant.

Can't get those fixes to work on my site so would be great if I can verify it on that. 

Thx

**Comment by pkarw** [comment]
No, they're not yet - I've committed the fixes to `develop` branch so they're not ready to deploy as there are many other unstable features. Check this commit please: https://github.com/DivanteLtd/vue-storefront/commit/8ea9b0aee1e8f63dedf18304da820296decbeba7

**Comment by jorkvist** [comment]
@pkarw thanks.

I have the changes implemented and deployed on ribatex.se but still the product page is broken when i reload it or browse to it on mobile. Is it something more i need todo?

**Comment by pkarw** [comment]
To be honest with you I have no idea how we can clear out the SW cache without explicitly clicking “Clear Storage” on these devices

**Comment by pkarw** [comment]
Would be great to think it off in case we’ll modify the service worker logic in the feature

@patzick @filrak any ideas?

I’ve got the exactly same issue on test.storefrontcloud.io and the only option was to clear storage :/ just after hitting this button everything started to work just great

**Comment by jorkvist** [comment]
Oh i see, i can also verify that the fix works for new browsers! I will try to implement a new cacheName.

**Comment by jorkvist** [comment]
FYI, the issue for me on the product page was the webshare function that was on the product page. Removed that one and now the product page works. I think it takes the wrong url and crashes.

**Comment by patzick** [comment]
@jorkvist thanks for information. Only removing it entirely solves this problem or is there something to improve to make it work?

cc @filrak - can you take a look? Maybe we shouldn't load it on first render?

**Comment by jorkvist** [comment]
@patzick, i had the old version of the webshare file, maybe the changes from today in 1.7.2 in that file fixed it. Haven't tested.
